v3/src/components/web-view/web-view-model.ts

```diff
@@ -11,12 +11,16 @@ export const WebViewModel = TileContentModel
     state: types.frozen<unknown>()
   })
   .volatile(self => ({
-    dataInteractiveController: undefined as iframePhone.IframePhoneRpcEndpoint | undefined
+    dataInteractiveController: undefined as iframePhone.IframePhoneRpcEndpoint | undefined,
+    isPlugin: false
   }))
   .actions(self => ({
     setDataInteractiveController(controller?: iframePhone.IframePhoneRpcEndpoint) {
       self.dataInteractiveController = controller
     },
+    setIsPlugin(_isPlugin: boolean) {
+      self.isPlugin = _isPlugin
+    },
```

Comment:
```suggestion
    setIsPlugin(isPlugin: boolean) {
      self.isPlugin = isPlugin
    },
```
Shouldn't need the underscores here.